src/test_lib_json/main.cpp

```diff
@@ -307,7 +307,21 @@ JSONTEST_FIXTURE(ValueTest, arrayIssue252) {
   }
   // JSONTEST_ASSERT_EQUAL(5, root["array"].size());
 }
+JSONTEST_FIXTURE(ValueTest, arrayIssue691) {
+  Json::Value array2;
+  array2.append(10);  // index 0
```

Comment:
Suggestion: Remove the comment and append a hint-like string value like "index0" instead of 10.